### DISCUSSION TITLE
AdvancedMarker custom icon not appearing

### DISCUSSION THREAD

**Discussion by Remarketeer** [visgl/react-google-maps#558] [Q&A]
I'd like to ask for some help!

So a week ago I left my maps page alone to work on other stuff, but suddenly my markers disappeared. For some context, I have an array that I load information from and use .map to create markers for each entry. After some trial and error, I realized that it just wasn't accepting image inputs, e.g <img src='' width=.../>. 

I've attempted to load the image on my other pages and that was fine. I've attempted to load Pin objects and that worked too, I've tried loading just a singular marker using an image source but it behaves the same way. Loading the image directly on the maps page also seems to work so the issue shouldn't be that.  I'm quite stumped as to where it could have gone wrong.

Currently the data seems to have been loaded into the map as I've printed it to the console, but the marker just isn't appearing.

**Comment by usefulthink** [comment]
Can you share a small code-sample of the jsx you use to create the markers? Something like this should work:

```tsx
const CustomMarker = ({position}) => {
  return (
    <AdvancedMarker position={position}>
      <img width={32} height={32} src={imageUrl} />
    </AdvancedMarker>
  );
};
```

**Reply by Remarketeer** [reply]
Sure! Here's the code snippet, they were working for quite some time, just...it suddenly failed.

```
{
latlng.map(item => (
<AdvancedMarker position={item} onClick={handleMarkerClick}>
    <img src={imgurl} width={50} height={35} /> 
  </AdvancedMarker>)) 
  }
```

I also tried placing one individual marker, which also failed
```
<AdvancedMarker
position={latlng[0]}>
       <img src={imgurl} width={50} height={35} /> 
  </AdvancedMarker>
```

**Reply by usefulthink** [reply]
ok, this sounds like a bug.
Can you check if it works as expected in version 1.1.3 (`npm install @vis.gl/react-google-maps@1.1.3`), that is before we updated the AdvancedMarker implementation.

**Reply by Remarketeer** [reply]
Hmmm, it still doesn't work after updating it to 1.1.3, not sure if something broke somewhere that prevents images from displaying...

**Reply by usefulthink** [reply]
I'll have to look into that in more detail. I created an issue here: https://github.com/visgl/react-google-maps/issues/560

**Comment by vaibhavugale** [comment]
It might possible that , your css is conflicting , in my case i have given  overflow  hidden to root in index.css , that is why marker's was not displaying

**Reply by leonardomartinsleo** [reply]
It works for me, thank you bro!
